sklearn/model_selection/_validation.py

```diff
@@ -1636,6 +1637,7 @@ def permutation_test_score(
     params = _check_params_groups_deprecation(fit_params, params, groups, "1.8")
 
     X, y, groups = indexable(X, y, groups)
+    params = params or {}
```

Comment:
This one seems redundant since we're calling `_check_params_groups_deprecation` just before, which already does this. I don't think params can be None here

Reply:
nice catch!

Reply:
Wouldn't it make sense to leave it there, because the call to _check_params_groups_deprecation will be removed later?

Reply:
we'd put it there when we remove that function. And we don't have to worry about it breaking code, cause the test will fail once we remove it.